### DISCUSSION TITLE
Drawing levels on top of each other

### DISCUSSION THREAD

**Discussion by mainrs** [Trouv/bevy_ecs_ldtk#387] [Q&A]
I am creating a tower where some floors have holes in them. I want the floor under the current one to be visible. 

Can this be done easily with this plugin? It looks like as soon as I switch the `LevelSelection`, everything is removed and the new entities are created.

**Comment by Trouv** [comment]
You can spawn multiple levels at once, either automatically by setting `load_level_neighbors` to `true` in `LdtkSettings`:
```rust
use bevy::prelude::*;
use bevy_ecs_ldtk::prelude::*;

fn main() {
    App::new()
        // other App builders
        .insert_resource(LevelSelection::index(0))
        .insert_resource(LdtkSettings {
            level_spawn_behavior: LevelSpawnBehavior::UseWorldTranslation {
                load_level_neighbors: true
            },
            ..default()
        })
        .run();
```

Or by not using `LevelSelection` at all and selecting multiple levels by their iid in the `LdtkWorldBundle`'s `LevelSet` component. In either case you should probably be using the `LevelSpawnBehavior::UseWorldTranslation` option in the above example.
